geometry_msgs/PointStamped bag_handle
float32 center_x
float32 center_y
